pybamm/models/submodels/thermal/isothermal/isothermal.py

```diff
@@ -23,7 +23,7 @@ def __init__(self, param):
 
     def get_fundamental_variables(self):
 
-        T_x_av = pybamm.PrimaryBroadcast(0, "current collector")
+        T_x_av = pybamm.PrimaryBroadcast(self.param.T_init, "current collector")
```

Comment:
Is this likely to affect other models other than the new basic ones?

Reply:
yeah, this was a bug fix